assets/agenda/reducers.ts

```diff
@@ -161,7 +170,10 @@ export default function agendaReducer(state: IAgendaState = initialState, action
     switch (action.type) {
 
     case RECIEVE_ITEMS:
```

Comment:
do you mind fixing the typo here taking advantage of this task? :)   
```suggestion
    case RECEiVE_ITEMS:
```